packages/manager/src/managers/telemetry/TelemetryManager.ts

```diff
@@ -92,6 +115,11 @@ export class TelemetryManager extends BaseManager {
 			payload.context.groupId.Repository = repository;
 		}
 
+		if (event === "page-view") {
+			payload.properties ||= {};
```

Comment:
Just double check, but I think `properties` is always defined (line 99), no need to initialize it again here